.gitmodules

```diff
@@ -5,7 +5,7 @@
   path = opendbc
   url = ../../commaai/opendbc.git
 [submodule "msgq"]
-  path = msgq
+	path = msgq_repo
```

Comment:
tab?

Reply:
This happens when you do `git mv` btw that's why I keep hitting it